types/react-beautiful-dnd/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
The code style is different from other libraries, if I'm right, it should only be `{ "extends": "dtslint/dt.json" }` in this file.